designer/client/src/components/graph/types/index.ts

```diff
@@ -0,0 +1,39 @@
+import { ProcessCounts } from "../../../reducers/graph";
```

Comment:
Do we need this folder structure? `designer/client/src/components/graph/types/index.ts`? Maybe designer/client/src/components/graph/types.ts will be enough here?

Reply:
I don't get you, why you want to change import type?

Reply:
If I understand correctly this green icon means that this file was added in this mr, right?
<img width="301" alt="image" src="https://github.com/TouK/nussknacker/assets/9945753/a2ee7df6-41d9-49d6-bd80-4c844185a0ef">

so I suggest not making it like `types/index.ts` but rather as `types.ts`. 

Is it clear now?

Reply:
yes, I created that file,  I think you are talking about `import { GraphProps } from "./types/index";` 
I changed to that` import { GraphProps } from "./types";`

Reply:
No, I'm talking about moving this file to the folder up and renaming it to `types.ts`, I don't see any profits from keeping types in the separate folder and `index.ts` file in this case.

Reply:
done